docker/docker-compose.objecttypes.yml

```diff
@@ -0,0 +1,20 @@
+version: '3'
+
+services:
+  db:
+    image: postgres
+    environment:
+      - POSTGRES_USER=${DB_USER:-objecttypes}
+      - POSTGRES_PASSWORD=${DB_PASSWORD:-objecttypes}
```

Comment:
I'd take the same structure/set up like in our root `docker-init-db.sql` - the reason for this is that you'll be created a database for objects and objecttypes each, but in the same DB cluster. You really don't want both django apps to look for the same tables in the same DB, that will cause problems.

Reply:
I created the init script but still got some weird errors (`FATAL: role "user" does not exist`)

I'm also unsure what the `open-forms-dev` network does and how it relates to the one in our docker compose.

I'm leaving this for now it takes too much of my time and I don't have all the knowledge so I'll come back to this later

Reply:
okay - some information:

`FATAL: role "user" does not exist` - in docker-compose, I basically tend to enable TRUST mode, it's not for prod anyway. So we let all the SQL be executed by the postgres superuser. That init script then ensures a DB user exists and creates the necessary database(s) owned by that user.

So in compose you typically want to have:
```
    environment:
      - POSTGRES_HOST_AUTH_METHOD=trust
```

The `open-forms-dev` network is so that we can spin up multiple compose files each from their own shell but all the services are on the same network. That way, we can make the root `docker-compose.yml` talk to the camunda from `docker/docker-compose.camunda.yml` etc. That essentially allows you to spin up *all the services* and have them be connected to use all of OF's features at once.